src/main/kotlin/com/exactpro/th2/check1/rule/nomessage/NoMessageCheckTask.kt

```diff
@@ -43,41 +43,40 @@ class NoMessageCheckTask(
     eventBatchRouter: MessageRouter<EventBatch>
 ) : AbstractCheckTask(ruleConfiguration, startTime, sessionKey, parentEventID, messageStream, eventBatchRouter) {
 
-    private val protoPreMessageFilter: RootMessageFilter = protoPreFilter.toRootMessageFilter()
-    private val messagePreFilter = SailfishFilter(
-        CONVERTER.fromProtoPreFilter(protoPreMessageFilter),
-        protoPreMessageFilter.toCompareSettings()
+    private var protoPreMessageFilter: RootMessageFilter? = protoPreFilter.toRootMessageFilter()
+    private var messagePreFilter: SailfishFilter? = SailfishFilter(
```

Comment:
I think you can make a DEFAULT instance for SailfishFilter and use it instead of `null`